src/components/Home-Page/JobsList/job.tsx

```diff
@@ -57,17 +37,27 @@ const Job: React.FC<JobsProps> = (props): JSX.Element => {
                 </div>
             </div>
             <div className={classes.jobFooter}>
-                <Button className={classes.jobBtn} startIcon={<VisibilityOutlinedIcon className={classes.btnIcon} />}> צפייה  </Button>
-                <Button 
-                    className={`${classes.jobBtn} ${isFavorite && classes.boldText}`}
-                    onClick={handleOnClickSaveButton}
-                    startIcon={getStarIcon()}
+                <Button
+                    className={classes.jobBtn} 
+                    startIcon={<VisibilityOutlinedIcon className={classes.btnIcon} />}
+                    onClick={() => setAreFullDetailsShown(true)}    
                 >
-                    {saveButtonText} 
+                    צפייה  
                 </Button>
+                <SaveJobButton isFavorite={isFavorite} handleClick={handleOnClickSaveButton} />
             </div>
+
+            <JobFullDetails
```

Comment:
Question, did you succeed to handle open only one ad?. It seems that you can open multiple ads at once and that can make troubles.
I think to move  the `JobFullDetails` to the `joblist` file and there handle the showFullDeatiles.
you can hold id ref to the full ad